akka-stream-tests/src/test/scala/akka/stream/io/TcpSpec.scala

```diff
@@ -430,7 +430,7 @@ class TcpSpec extends StreamSpec("akka.stream.materializer.subscription-timeout.
           .run()
 
       // make sure that the server has bound to the socket
-      val binding = Await.result(bindingFuture, 100.millis)
+      val binding = Await.result(bindingFuture, 1.second)
```

Comment:
* `remaining` is first choice (requires `within` block)
* `remainingOrDefault`is second choice
* `3.seconds` is third choice if not using testkit
* lower must come with a very good reason

Read up on `remaining` and friends in https://github.com/akka/akka/blob/master/akka-testkit/src/main/scala/akka/testkit/TestKit.scala